scripts/styleguide.config.js

```diff
@@ -1,4 +1,6 @@
 const path = require('path');
+// const typescriptDocGen = require('react-docgen-typescript');
```

Comment:
revert, but without this styleguide cannot detect props iirc. But thats fine, storybook will be taking care of this now on.